<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>303</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DB전송프로그램</string>
  </property>
  <widget class="QPushButton" name="imgLoadBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지불러오기</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputX1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputY1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputX2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputY2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputA">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>a 값</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>b 값</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputCctvName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>CCTV 이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>CCTV PORT</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputCctvPort">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>31</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="calcBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>계산하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>DB 전송</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>x1 값</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>y1 값</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>y2 값</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>x2 값</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>imgLoadBtn</tabstop>
  <tabstop>outputX1</tabstop>
  <tabstop>outputY1</tabstop>
  <tabstop>outputX2</tabstop>
  <tabstop>outputY2</tabstop>
  <tabstop>calcBtn</tabstop>
  <tabstop>outputA</tabstop>
  <tabstop>outputB</tabstop>
  <tabstop>inputCctvName</tabstop>
  <tabstop>inputCctvPort</tabstop>
  <tabstop>sendBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
